Arena.toml

```diff
@@ -1,358 +1,216 @@
-version = "v1"
```

Comment:
Do we need the version identifier? I don't know what the original intent was.

Reply:
The `version` here wasn't the configuration file format version, but the WDL version to parse with.

Since the new parser implementation responds to what is specified in the document and doesn't require specifying the version upfront, this configuration value is no longer needed.

Reply:
Thanks for the clarification. I'd rather parse it out of the file themselves, so a clear improvement.